### ISSUE TITLE
Not working?

### ISSUE THREAD

**Issue by tgozed** [BurntSushi/nflvid#15]
I've not used this package, but the script I wrote doesn't work this season - I assume you're finding the same problem?

**Comment by BurntSushi** [comment]
Can you show the commands you're running and the output you get?

**Comment by tgozed** [comment]
I have a PHP script that generates game URLs, specifically when trying to grab last nights HoF game I'm getting this in FFmpeg;

Unable to open key file http://nlsk.neulion.com/nlsk1/hls/securekey?id=82&url=/nlds_vod/nfl/vod/2015/08/09/56759/1_56759_PIT_MIN_2015_h_whole_1_4500.mp4/19700101000000.m3u8.key
[hls,applehttp @ 0xacaaf00] Error when loading first segment 'http://nlds82.cdnl3nl.neulion.com/nlds_vod/nfl/vod/2015/08/09/56759/1_56759_PIT_MIN_2015_h_whole_1_4500_000000.ts'
http://nlds82.cdnl3nl.neulion.com/nlds_vod/nfl/vod/2015/08/09/56759/1_56759_PIT_MIN_2015_h_whole_1_4500.mp4.m3u8: Invalid data found when processing input

**Comment by BurntSushi** [comment]
Can you please provide the `nflvid` command that you're running so that I can attempt to reproduce the issue?

That HLS stream doesn't appear to work when I field it into `vlc`. I wonder if the NFL finally shut this down.

**Comment by tgozed** [comment]
Sorry, I've not actually used nflvid (although I did use your code when I first made my script  a couple of years ago) 

The error I'm getting is just to say neulion now seems to need auth keys to pull the streams. So I think they have shut this down. I'll just have to hope someone uploads all the pats preseason games to bittorrent/usenet cause they don't air them in the UK.

You should be able to produce an error with any nflvid command.

**Comment by gainesc** [comment]
You should see the same error on any of the nflvid commands as well for this season. I've been having similar issues.

**Comment by Trakcf** [comment]
Does coaches film work for preseason games?  I can pull prior year's regular season but it doesn't seem to work for any of the preseasons.

**Comment by gainesc** [comment]
Not sure if coaches film has ever worked for preseason games.

**Comment by andr3w321** [comment]
Is this working for anyone?  Just tried to use for the first time 

```
$ .local/bin/nflvid-footage --dry-run --season 2014 --teams NE -- ~/nfl-footage/

No games matched your search criteria.
```

**Comment by BurntSushi** [comment]
Yes, it's working fine for me. Make sure you don't already have downloaded files in your `~/nfl-footage/` directory. `nflvid-footage` won't re-download games that already exist on disk.

**Comment by iversond** [comment]
I received a similar error as above when trying to download a game. I added 2015 to the season list in `nfl-footage`. This was the command I used:

    nflvid-footage --season 2015 --weeks 1 --teams MIN --condensed -- ~/Desktop/

This was the error message I received:

    [http @ 0x7f98f3614a20] HTTP error 403 Forbidden
    Unable to open key file http://nlsk.neulion.com/nlsk1/hls/securekey?id=82&url=/nlds_vod/nfl/vod/2015/09/14/56518/2_56518_min_sf_2015_h_snap2w_1_1600.mp4/19700101000000.m3u8.key
    [hls,applehttp @ 0x7f98f3826e00] Error when loading first segment 'http://nlds82.cdnl3nl.neulion.com/nlds_vod/nfl/vod/2015/09/14/56518/2_56518_min_sf_2015_h_snap2w_1_1600_000000.ts'
   http://nlds82.cdnl3nl.neulion.com/nlds_vod/nfl/vod/2015/09/14/56518/2_56518_min_sf_2015_h_snap2w_1_1600.mp4.m3u8: Invalid data found when processing input

**Comment by nexxai** [comment]
I'm having the same issues as @iversond is with the Forbidden thing.  Just wondering if any progress had been made with that.

**Comment by nexxai** [comment]
After thinking about it more, shouldn't the coach's film need the same key?  I don't quite know enough python to figure out what the difference might be, but shouldn't they both be calling the same function to get the key to begin with?

**Comment by nrnrnr** [comment]
Just to confirm, here's a "me, too" report with the command and the problem:

````
: nr@homedog 10719 ; nflvid-footage --season 2014 --teams NE --weeks 5 --broadcast /red1/nflvid
These are the games that match your search criteria:
(Season: 2014, Week: 5, CIN at NE)
Are you sure you want to start downloading? [y/n] y
Downloading game 2014100512 (Season: 2014, Week: 5, CIN at NE)
Could not run 'ffmpeg -timeout 120 -i http://nlds84.cdnl3nl.neulion.com/nlds_vod/nfl/vod/2014/10/05/56244/2_56244_cin_ne_2014_h_whole_1_1600.mp4.m3u8 -absf aac_adtstoasc -acodec copy -vcodec copy /red1/nflvid/2014100512.mp4' (exit code 1):
   avconv version 11.4-6:11.4-1~deb8u1, Copyright (c) 2000-2014 the Libav developers
     built on Jun  4 2015 19:43:21 with gcc 4.9.2 (Debian 4.9.2-10)
   [http @ 0x888daa0] HTTP error 403 Forbidden
   Unable to open key file http://nlsk.neulion.com/nlsk1/hls/securekey?id=84&url=/nlds_vod/nfl/vod/2014/10/05/56244/2_56244_cin_ne_2014_h_whole_1_1600.mp4/19700101000000.m3u8.key
   http://nlds84.cdnl3nl.neulion.com/nlds_vod/nfl/vod/2014/10/05/56244/2_56244_cin_ne_2014_h_whole_1_1600.mp4.m3u8: Invalid data found when processing input
FAILED to download game (Season: 2014, Week: 5, CIN at NE)
````

**Comment by nexxai** [comment]
Is there a way I can force the key file in if I can get it from another source?

**Comment by chrisgaines** [comment]
@nexxai have you had any luck getting it to work? I would love to get a hold of the coaches film for week 10 last season.
